src/main/java/com/commercetools/sync/CliRunner.java

```diff
@@ -0,0 +1,152 @@
+package com.commercetools.sync;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.CompletionStage;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+public class CliRunner {
+  static final String SYNC_MODULE_OPTION_SHORT = "s";
```

Comment:
It would be nice to separate `SYNC_MODULE_OPTION` constants to an enum. Also, we can move some messages(AVAILABLE_OPTIONS) to this enum. wdyt?

Reply:
why is it better?